core/App/ehsm_provider.h

```diff
@@ -324,4 +325,16 @@ apikey -- 32byte random number
 */
 ehsm_status_t Enroll(ehsm_data_t *appid, ehsm_data_t *apikey);
 
+/*
+Description:
+Generate HMAC(SHA-256) of payload with given apikey
+Input:
+cmk -- A symmetric cmk
+apikey -- 32byte random number (in plaintext, not encrypted)
```

Comment:
it should be in ciphertext

Reply:
Thanks for review! This is resolved in the latest commit.